assets/js/main.js

```diff
@@ -17,4 +18,10 @@ async function getQuotes() {
   tagsList.innerHTML = listHtml;
 }
 
-getQuotes();
+getQuotes(); 
+
+refreshButton.addEventListener("click", function() {
+  getQuotes();
+});
+
+ 
```

Comment:
There should be only one empty line at the end of the file. Please remove line 27.